frontend/src/containers/OnBoarding.js

```diff
@@ -99,18 +105,22 @@ export class OnBoarding extends Component {
 
     return validateSchema(githubForm.attributes, githubSchema)
       .then(() => createGroupFromGithubRepo(payload, attr.token))
-      .then(() => this.setState({step: 5}))
+      .then(() => this.setState({step: 4}))
```

Comment:
Oops I meant to select line 120 or so. But I think that flow works fine because you can select another repo.